src/main/java/frc/team88/swerve/motion/kinematics/ForwardKinematics.java

```diff
@@ -103,6 +104,11 @@ public ForwardKinematics(SwerveModule... modules) {
     SingularValueDecomposition svd = new SingularValueDecomposition(inverseKinematics);
     DecompositionSolver solver = svd.getSolver();
     forwardKinematics = solver.getInverse();
+    RealMatrixFormat TABLE_FORMAT = new RealMatrixFormat("", "", "", "\n", "", ", ");
+    System.out.println("inverseKinematics:");
+    System.out.println(TABLE_FORMAT.format(inverseKinematics));
+    System.out.println("forwardKinematics:");
+    System.out.println(TABLE_FORMAT.format(forwardKinematics));
```

Comment:
If we leave these print statements in, they will show up for everyone that uses this library, and I don't think we want that